### PR TITLE
IODA-128: Enforce PSR-12

### PR DESCRIPTION
**Pull request by bartfeenstra**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -62,19 +58,34 @@
...
-            "@auto-scripts"
+            "@auto-scripts",
+            "npm install"
         ],
         "post-update-cmd": [
-            "@auto-scripts"
+            "@auto-scripts",
+            "npm update"
+        ],
+        "test": [
```
Comment (mdbudd) [inline composer.json:68]: Nice!

File: composer.json [hunk 2]
```diff
@@ -39,14 +40,9 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "App\\Tests\\": "tests/"
+            "App\\Tests\\": "tests/src"
```
Comment (mdbudd) [inline composer.json:43]: Noted, although not default symfony way. Seems like tests naturally sit outside of app but not overly contesting this.

File: phpunit.xml.dist [hunk 3]
```diff
@@ -22,10 +22,10 @@
         </testsuite>
 
         <testsuite name="integration">
-            <directory>tests/integration</directory>
+            <directory>tests/src/Integration</directory>
```
Comment (mdbudd) [inline phpunit.xml.dist:25]: Please see above comment.

**Review by mdbudd — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (bartfeenstra) [reply composer.json:43]: You're right. Depends on where you put any test assets. I you put them somewhere outside `./tests` then yes. I made the assumption we'd put them in `./tests`, but that can be changed.

**Review by baldygeezer — APPROVED** [review]
I am happy with this. At this stage details about test organisation and directory structure shouldn't be a massive worry. We will probably need to refactor a few times to arrive at an optimal solution as things grow

**Pull request merged**